### PR TITLE
Add support for launching Geth and Nimbus-eth1 nodes in launch_local_testnet

### PR DESCRIPTION
**Pull request by zah**
Authored by @hmel

### CODE REVIEW

File: scripts/launch_local_testnet.sh [hunk 1]
```diff
@@ -72,7 +82,20 @@ ETH2_DOCKER_IMAGE=""
 REMOTE_SIGNER_NODES=0
 REMOTE_SIGNER_THRESHOLD=1
 REMOTE_VALIDATORS_COUNT=0
-LC_NODES=1
+LC_NODES=0
```
Comment (etan-status) [inline scripts/launch_local_testnet.sh]: hm?

Comment (zah) [reply scripts/launch_local_testnet.sh]: I'm not sure how this came to be, but it looks like the network is failing to finalize when there are some light nodes under the new regime here. I've pushed `LC_NODES=1`, so you can take a look at the logs.

Comment (etan-status) [reply scripts/launch_local_testnet.sh]: Seems like it finalizes properly.

File: scripts/launch_local_testnet.sh [hunk 2]
```diff
@@ -740,13 +928,15 @@ if [ "$LC_NODES" -ge "1" ]; then
...
-        "${JQ_BINARY}" -r '.data.epoch')"
+        tee -a curl_result.txt | "${JQ_BINARY}" -r '.data.epoch')"
     if [ "${CURRENT_FORK_EPOCH}" -ge "${ALTAIR_FORK_EPOCH}" ]; then
       break
     fi
     sleep 1
   done
 
+  log "After ALTAIR_FORK_EPOCH"
+
```
Comment (etan-status) [inline scripts/launch_local_testnet.sh:939]: There is already a log one line below this

File: scripts/launch_local_testnet.sh (hunk 1 above)
Comment (zah) [reply scripts/launch_local_testnet.sh]: It fails locally, but alright, we can keep investigating this after merging.

**Pull request merged**